sway-lib-std/src/u128.sw

```diff
@@ -49,7 +49,7 @@ impl core::ops::Ord for U128 {
 // impl core::ops::OrdEq for U128 {
 // }
 
-fn disable_overflow() {
+pub fn disable_overflow() {
```

Comment:
Refactor these into another modular in a separate PR, that can be merged into `master` first.

Reply:
yep, will do as mentioned in the description. going to convert this to a draft and land some cleanup PRs first then.